.github/workflows/pull_request.yml

```diff
@@ -157,5 +157,4 @@ jobs:
         docker exec -t test ls
         docker exec -t test bundle update
         docker exec -t test bundle exec rake
-        docker exec -t test bundle exec rake suites_run
```

Comment:
Unable to isolate the _OpenStudio-Standards_ gem issue (prototype test suite) for now. OK with skipping the _prototype_ test suite altogether when it comes to a _bleeding edge_ version of the OpenStudio/SDK (v3.9.0 was released last week). Can certainly wait for the next release.